<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addItem</class>
 <widget class="QWidget" name="addItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>801</width>
     <height>601</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="itemdetailPage">
    <widget class="QLabel" name="label_image">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>160</y>
       <width>221</width>
       <height>231</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>801</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_logo_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="info_item">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>431</width>
       <height>371</height>
      </rect>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="lineEdit_name">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px; padding: 2px; border: 1px solid gray; </string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>21</x>
        <y>41</y>
        <width>39</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Name:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Manufacturer:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Starting price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>140</y>
        <width>58</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>BIN price:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Duration</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>200</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Description:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_manufacturer">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>80</y>
        <width>221</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px; padding: 2px; border: 1px solid gray; 
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_startingPrice">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>110</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px; padding: 2px; border: 1px solid gray; </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_binprice">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>140</y>
        <width>111</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px; padding: 2px; border: 1px solid gray; </string>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit_auctiontime">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>121</width>
        <height>22</height>
       </rect>
      </property>
      <property name="currentSection">
       <enum>QDateTimeEdit::HourSection</enum>
      </property>
      <property name="displayFormat">
       <string>HH:mm:ss</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_description">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>210</y>
        <width>291</width>
        <height>141</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px; padding: 2px; border: 1px solid gray; </string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_back">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>70</y>
       <width>151</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>24</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Sell an item</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_choosePic">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>430</y>
       <width>141</width>
       <height>32</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="text">
      <string>Choose a picture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_cancel">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>520</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

	background-color:rgb(255, 100, 100);
 border-radius: 5px;
border: 1px solid;
 }</string>
     </property>
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_save">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>520</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{

	background-color: rgba(105, 183, 255, 176);
 border-radius: 5px;
border: 1px solid;
 }</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
